doc/source/whatsnew/v0.22.0.txt

```diff
@@ -139,6 +139,7 @@ Other Enhancements
 - :func:`read_excel()` has gained the ``nrows`` parameter (:issue:`16645`)
 - :func:``DataFrame.to_json`` and ``Series.to_json`` now accept an ``index`` argument which allows the user to exclude the index from the JSON output (:issue:`17394`)
 - ``IntervalIndex.to_tuples()`` has gained the ``na_tuple`` parameter to control whether NA is returned as a tuple of NA, or NA itself (:issue:`18756`)
+- ``Categorical.rename_categories`` can how take a callable as its argument (:issue:`18862`)
```

Comment:
use ``:func:`Categorical.rename_categories` `` here

Reply:
how -> now

Reply:
Also, we don't have `Categorical` in the API docs yet, so that won't work.

    :meth:`Series.cat.rename_categories`

may work (though it applies to `CategoricalIndex` too).

Reply:
actually let's also test this for a CI and a Series.cat.rename_categories

Reply:
Not sure I understand this about CI. I assume that will be a seperate PR? else I'd appreciate guidance.

Do you mean test ``Series.cat.rename_categories``? Isn't it a bit odd  as that's a delegation from the ``Categorical`` namespace (not at the proper computer ATM, but that's how I understand it)?

Reply:
CI = CategoricalIndex, sure its separate but makes sense to do it here (it should just work). Series.cat....IS a delegation, but we do test things there as well (for example to make sure things are passed thru). See what we have for testing and add as appropriate (or indicate that we are not testing except for basic functionaility).

Reply:
Ok, I understood CI as Continuous integration. Sure, I can take a look at adding a test for  CategoricalIndex,